requirements.txt

```diff
@@ -0,0 +1,10 @@
+# Requirements for building PEPs with Sphinx
+sphinx >= 3.5
+docutils >= 0.16
+
+# # For RSS
+# feedgen >= 0.9.0  # For RSS feed
```

Comment:
Why have you commented on the following lines?

Reply:
Half laziness in reserving the spaces for the RSS piece of work - should #1934 be merged the lines would be subsequently uncommented. If easier I could remove from this PR just for clarity?

Reply:
Could you remove these lines and add them later in #1934? That way the commit is clear